docker/server/scripts/server_common.sh

```diff
@@ -741,3 +741,49 @@ detect_amc_board()
          echo "Auto-detection of presence, type and version of AMC boards disabled."
     fi
 }
+
+# This function calls all the initialization routines. It returns a list of extra
+# arguments which are generated by some if the routines that auto-detect hardware versions.
+# The first argument indicates the variable to store the result list of extra argument.
+initialize()
+{
+    # The first argument points to a variable name to store the resulting argument list
+    local __result_args=$1
+
+    # This will hold the list of extra arguments
+    local __extra_args
+
+    # Validate the selected communication type
+    validateCommType
+
+    # Validate the selected slot number
+    validateSlotNumber
+
+    # Get FPGA IP address
+    getFpgaIpAddr
+
+    # Look for pyrogue files
+    findPyrogueFiles
+
+    # Firmware version checking
+    checkFW
+
+    # Do a hard boot, if requested
+    hardBoot
+
+    # Auto-detect hardware type
+    ## Detect type of AMCs, and get specific server startup arguments
+    ## for each specific type and add them to the list of arguments
+    local __extar_amcs_args
```

Comment:
Is this a typo of extra?

Reply:
Yeap. Thanks for catching it @jlashner . Fortunately (or unfortunately?) all reference to the variable had the same typo so the final result was correct. I fixed it anyways.